openmc/surface.py

```diff
@@ -23,8 +23,7 @@ def reset_auto_surface_id():
 
 
 class Surface(object):
-    """A two-dimensional surface that can be used define regions of space with an
-    associated boundary condition.
+    """A two-dimensional surface with an associated boundary condition.
```

Comment:
Perhaps "three-dimensional" is more appropriate since we have `Sphere` and `Cone`, and all of others are also 3D.